README.md

```diff
@@ -60,11 +60,117 @@ You can see a big picture of the roadmap here: [**ROADMAP**][roadmap]
 - [Wallet types](https://github.com/kommitters/kadena.ex/issues/18)
 - [Kadena Crypto](https://github.com/kommitters/kadena.ex/issues/51)
 - [Kadena Pact](https://github.com/kommitters/kadena.ex/issues/55)
+- [Kadena Pact Commands Builder](https://github.com/kommitters/kadena.ex/issues/131)
```

Comment:
@Odraxs @EdwinGuayacan Actually this is not done, since the implementation of the Execution command is missing.